velox/exec/tests/RowContainerTest.cpp

```diff
@@ -1128,3 +1130,56 @@ TEST_F(RowContainerTest, probedFlag) {
           {true, true, true, true, std::nullopt, true}),
       result);
 }
+
+template <typename T>
+inline T* valueAt(char* row, int32_t offset) {
+  return reinterpret_cast<T*>(row + offset);
+}
+
+TEST_F(RowContainerTest, shareAndClear) {
+  auto initial = FLAGS_velox_check_row_container_free;
```

Comment:
I think you can add gflags::FlagSaver flagSaver_; in RowContainerTest class to handle the gflag resetting